zipkin-server/src/main/java/zipkin2/server/internal/ZipkinQueryApiV2.java

```diff
@@ -156,6 +157,7 @@ public AggregatedHttpMessage getAutocompleteValues(@Param("key") String key) thr
    * active use, and active use usually implies more than 3 services.
    */
   AggregatedHttpMessage maybeCacheNames(boolean shouldCacheControl, List<String> values) {
+    Collections.sort(values);
```

Comment:
Why sort them in the server and in the UI? I think just sorting in the UI is fine until we have a reason to sort them in the API response.